test/shim/test_sequencer.py

```diff
@@ -490,6 +580,31 @@ def test_shim_sequencer_rt_larger_coil(self, nii_fieldmap, json_data, nii_anat,
 
         assert np.all(currents_static.shape == (20, 3))
 
+    def test_shim_sequencer_rt_qp_larger_coil(self, nii_fieldmap, json_data, nii_anat, nii_mask_static,
```

Comment:
I think we should only have a single test for larger coils as they take more time. Since quadprog will be the default, we should use that.

Reply:
Yes, sure